ansible/roles/automation/openclose_pr/tasks/main.yml

```diff
@@ -0,0 +1,82 @@
+---
+- name: install packages dependencies
+  dnf:
+    name: "{{ item }}"
+    state: present
+  with_items:
+    - python3-github3py
+    - PyYAML
+
+- name: install pip dependencies
+  pip:
+    name: "{{ item }}"
+    executable: pip3
+  with_items:
+    - CacheControl
+    - gitpython
+
+- name: create base directory
+  file:
+    path: "{{ basedir }}"
+    state: directory
+    owner: root
+    group: root
+    mode: 0700
+
+- name: create configuration file
+  template:
+    src: config.yml
+    dest: "{{ basedir }}/config.yml"
+
+- name: copy openclose_pr.py script
+  copy:
+    src: "{{ playbook_dir }}/../github/open_close_pr.py"
+    dest: "{{ basedir }}"
+
+- name: copy custom freeipa-pr-ci.yaml file
+  copy:
+    src: "{{ prci_config }}"
+    dest: "{{ basedir }}/{{ identifier }}.yaml"
+
+- name: copy git ssh key
+  copy:
+    src: "{{ git_sshkey }}"
+    dest: "/root/.ssh/git_ssh_key"
+    mode: 0600
+    owner: root
+    group: root
+
+- name: configure ssh to use the right ssh key when using git
+  blockinfile:
+    path: /root/.ssh/config
+    block: |
+          Host openclose_tool.com
```

Comment:
Is there a reason to use a non-existent imaginary hostname? Why not `Host github.com`?

Reply:
I have tried to keep the openclose tool as isolated as possible. So, if a machine configured with this tool is used by another tool for another purpose, the ssh key configuration would not become confusing (assuming that all tools should use the same key)

Reply:
Ugh. Using imaginary domain name that might exist creates way more confusion.

If you want complete isolation on command-by-command basis, the do it properly with [`GIT_SSH_COMMAND`](https://stackoverflow.com/a/29754018/6081607).

If it's no necessary, you can use `Host github.com` to set defaults.

Or, just create a hostname like you did, but please do not use something that looks like actual domain name (e.g. `Host github-pr_tool` is fine).

Reply:
Ah ok, the problem you see is the hostname not with the use of it. Ok, cool. I'll change it.